app/routes/wow.out-of-stock.tsx

```
@@ -0,0 +1,287 @@
+const validateInput = z.object({
+  salesPerDay: z.string().transform((value) => parseFloat(value)),
+  avgPrice: parseStringToNumber,
+  minMarketValue: parseStringToNumber,
+  populationWP: parseStringToNumber,
+  populationBlizz: parseStringToNumber,
+  rankingWP: parseStringToNumber,
+  expansionNumber: parseStringToNumber
+})
+
+type ActionResponseType =
+  | {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)